circle.yml

```diff
@@ -113,7 +120,31 @@ jobs:
       - store_artifacts:
           path: ~/.npm/_logs
 
-  "kitchensink-chrome":
+  kitchensink-windows:
+    executor: win
+    steps:
+      - prepare
+      - run:
+          command: npm run test-if -- --repo cypress-example-kitchensink --command test:ci:record:windows
```

Comment:
shouldn't this be in `firefox` not electron?

Reply:
I wanted to start with Chrome, since that's what failed on Appveyor, but turns out there are no browsers pre-installed at all.